src/components/Global.js

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components';
```

Comment:
Good job having the Global.js - I have tried this but didn't really figure it out yet. Good job with your and I'll take some inspiration from this one :)